Makefile

```diff
@@ -53,7 +56,9 @@ check-coverage: test-unit test-integration  ## Unit tests and integration tests
 test-e2e: ## Run e2e tests - requires running OLS server
 	@echo "Running e2e tests..."
 	@echo "Reports will be written to ${ARTIFACT_DIR}"
-	python -m pytest tests/e2e -o junit_suite_name="${SUITE_ID}" -m "${TEST_TAGS}" --junit-prefix="${SUITE_ID}" --junit-xml="${ARTIFACT_DIR}/junit_e2e_${SUITE_ID}.xml"
+	# Currently eval model is gpt always, till we add QnA for different model.
+	python -m pytest tests/e2e -o junit_suite_name="${SUITE_ID}" -m "${TEST_TAGS}" --junit-prefix="${SUITE_ID}" --junit-xml="${ARTIFACT_DIR}/junit_e2e_${SUITE_ID}.xml" \
+	--eval_model "gpt" --eval_threshold ${EVAL_THRESHOLD}
```

Comment:
see comments above about defaulting threshold and getting the model from existing env vars